lib/api/apiUtils/bucket/bucketDeletion.js

```diff
@@ -110,7 +110,7 @@ export function deleteBucket(bucketMD, bucketName, canonicalID, log, cb) {
 
     return async.waterfall([
         function checkForObjectsStep(next) {
-            return metadata.listObject(bucketName, null, null, null, 1, log,
+            return metadata.listObject(bucketName, { maxKeys: 1 }, log,
```

Comment:
This is in the context of deleting a bucket. In order to delete a bucket, we need to ensure that it doesn't contain any object, and listing is a way to check that. I think using `maxKeys: 1` here is an optimization; you can also use the max threshold, but it just wastes disk ios and bandwidth.